src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+/**
+ * The class to store tasks into a text file.
+ */
+public class Storage {
+    private String filePath;
+
+    /**
+     * Storage constructor.
+     *
+     * @param filePath The relative path of the text file that tasks are saved into.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Saves the TaskList into the text file.
+     * Prints the error message if there is one.
+     *
+     * @param tl TaskList that contains objects of type Task.
+     */
+    public void saveTaskList(TaskList tl) {
+        try {
+            FileOutputStream fos = new FileOutputStream(this.filePath);
+            ObjectOutputStream oos = new ObjectOutputStream(fos);
+            oos.writeObject(tl);
+            oos.close();
+            fos.close();
+            System.out.println("    Saved");
+        } catch (Exception e) {
```

Comment:
Maybe try not to shorten the variables name too much?

fos -> fileOutput
oos -> objectOutput

The original variable names are concise but don't really tell the other viewers what they are.